actions/soak_helpers.go

```diff
@@ -16,8 +16,9 @@ import (
 
 // BuildGoTests builds the go tests using native go cross-compilation to run, and returns a path to the test executable
 // along with its size in bytes.
-//  Note: currentProjectRootPath and currentSoakTestRootPath are not interchangeable with utils.ProjectRoot and utils.SoakRoot
-//  when running in outside repositories. Keep an eye on when you need paths leading to this go package vs the current running project.
+//
```

Comment:
```suggestion
```

Reply:
go fmt automatically adds this and if it isn't set like this then the linter gets angry.